src/var.cpp

```diff
@@ -172,6 +172,94 @@ const char *var_kind_name[(int) VarKind::Count] {
     "extract"
 };
 
+const bool var_kind_fp16_unsupported_cuda[(int) VarKind::Count] {
```

Comment:
would it not be easier to understand to encode the inverse? (i.e., `fp16_supported`)